packages/theme/button.ts

```diff
@@ -36,13 +36,14 @@ export default StyleSheet.create({
     borderRadius: 100
   },
   iconRound: {
-    marginTop: 2,
+    marginTop: -2,
```

Comment:
I experimented with this and I think we should just remove the line and that will center the icon